### ISSUE TITLE
Factor out some UI logic

### ISSUE THREAD

**Issue by ratijas** [Nit31/obsidian-gpt-text-processing#6]
https://github.com/Nit31/obsidian-gpt-text-processing/blob/15cf3d40aea4d59bc5bb3ec05c607169b29c9b83/src/ui/process-modal.ts#L68-L69

could be replaced with a call to `updateButtonState`

**Comment by ratijas** [comment]
Also, triple dot should be replaced with an ellipsis for accessibility reasons: `…`
